### PR TITLE
Create initial package

### PR DESCRIPTION
**Pull request by lhursh**

### CODE REVIEW

File: ListrakSDK-iOS.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "1.0.0":  "https://github.com/listrak/Listrak-iOS-SDK/blob/create-initial-package/ListrakSDK-iOS.zip?raw=true"
```
Comment (lhursh) [inline ListrakSDK-iOS.json]: - [x] Update this to point to master before merge

**Review by ZacharyGroff — APPROVED** [review]
lgtm

**Review by josh-riggs-listrak — APPROVED** [review]

**Review by ZacharyGroff — APPROVED** [review]
lgtm

**Review by dt9208 — APPROVED** [review]

**Review by ZacharyGroff — APPROVED** [review]
lgtm

**Pull request merged**